cmd/argo/commands/list.go

```diff
@@ -64,6 +64,34 @@ func NewListCommand() *cobra.Command {
 	command := &cobra.Command{
 		Use:   "list",
 		Short: "list workflows",
+		Example: `# List all workflows:
+  argo list
+
+# Show workflows from all namespaces:
+  argo list -A
+
+# List all running workflows:
+  argo list --running
+
+# List all completed workflows:
+  argo list --completed
+
+# List workflows that finished more than 2h ago:
+  argo list --older 2h
+
+# List workflows with more information (such as parameters):
+  argo list -o wide
+
+# List workflows in YAML format:
+  argo list -o yaml
+
+# List workflows that have both labels:
+  argo list -l label1=value1,label2=value2
+
+# List workflows created within the last 10m:
+  argo list --since 10m
```

Comment:
this still needs to be moved above `--older`, per my [previous comment](https://github.com/argoproj/argo-workflows/pull/12076#discussion_r1372078137)